lang/csharp/src/apache/main/Generic/GenericWriter.cs

```diff
@@ -29,59 +30,63 @@ namespace Avro.Generic
     public delegate void Writer<T>(T t);
 
     /// <summary>
-    /// A typesafe wrapper around DefaultWriter. While a specific object of DefaultWriter
+    /// A type-safe wrapper around DefaultWriter. While a specific object of DefaultWriter
     /// allows the client to serialize a generic type, an object of this class allows
     /// only a single type of object to be serialized through it.
     /// </summary>
     /// <typeparam name="T">The type of object to be serialized.</typeparam>
+    /// <seealso cref="DatumWriter&lt;T&gt;" />
     public class GenericWriter<T> : DatumWriter<T>
     {
-        private readonly DefaultWriter writer;
+        private readonly DefaultWriter _writer;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="GenericWriter{T}"/> class.
+        /// Initializes a new instance of the <see cref="GenericWriter{T}" /> class.
         /// </summary>
         /// <param name="schema">Schema to use when writing.</param>
         public GenericWriter(Schema schema) : this(new DefaultWriter(schema))
         {
-
         }
 
         /// <inheritdoc/>
-        public Schema Schema { get { return writer.Schema; } }
+        public Schema Schema
+        { get { return _writer.Schema; } }
```

Comment:
Maybe just "public Schema Schema => _writer.Schema;``, since it is a getter only?

Reply:
I've been trying to remember the rule that requests you do that.  I used to do it for any single line method as well.

Reply:
Found it:
csharp_style_expression_bodied_methods = true
csharp_style_expression_bodied_constructors = true
csharp_style_expression_bodied_operators = true
csharp_style_expression_bodied_properties = true
csharp_style_expression_bodied_indexers = true
csharp_style_expression_bodied_accessors = true
csharp_style_expression_bodied_lambdas = true
csharp_style_expression_bodied_local_functions = true

Let me dig into why I'm not getting a message.

Reply:
It is most likely because of `dotnet_diagnostic.IDE0021` - `dotnet_diagnostic.IDE002x`. Many settings can be controlled by  editorconfig and the ruleset file. However the ruleset file will have higher precedence. IMO settings which are controllable by ruleset files should be in the ruleset file only, because it is easier to track back the actual warning generated by the compiler. editorconfig should have only the settings which generate the 'Fix formatting' or other kind of umbrella warnings.

Reply:
It appears that my local user settings are overwriting the editorconfig settings.

Reply:
One moment need to fix

Reply:
Should be fixed now